doc/source/howtos/upgrade_guide.rst

```diff
@@ -117,3 +120,17 @@ To upgrade the cluster, follow these steps for each node:
 #. Repeat this procedure for the next node in the cluster
    until you upgrade all nodes.
 
+.. note:: It is important that when rejoining, the node should synchronize using
+   :term:`IST`. If for some reason :term:`SST` syncronization is used
+   instead, there is a problem, as the node will use data-directory from
+   the previous version, and newer version binaries would throw a lot of
+   errors about wrong structure. This :term:`SST`-related problem is
+   garanteed in case of major upgrade, and is possible in case of minor
+   upgrade also.
+
+   If this problem occurs, it can be solved with the following steps: 
```

Comment:
Rather than saying problem occurs let's simply put that as the user need to do it whenever the node joins using SST. (Always needed step).

Reply:
As above